v2/pkg/genruntime/wellknown-resource-reference.go

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package genruntime
+
+// WellknownResourceReference is a variation of ResourceReference that also permits the referenced resource to be
+// identified by a well known name.
```

Comment:
I don't think this file should be here -- I thought we already had it?

Also, file name is wrong again =)